Source/WebCore/platform/graphics/ImageBufferContextSwitcher.cpp

```diff
@@ -82,9 +82,11 @@ void ImageBufferContextSwitcher::endClipAndDrawSourceImage(GraphicsContext& dest
 
 void ImageBufferContextSwitcher::endDrawSourceImage(GraphicsContext& destinationContext, const DestinationColorSpace& colorSpace)
 {
+    ASSERT_IMPLIES(m_filter, m_sourceImage);
+
     if (!m_filter) {
-        ASSERT(m_sourceImage);
-        destinationContext.drawImageBuffer(*m_sourceImage, m_sourceImageRect, { destinationContext.compositeOperation(), destinationContext.blendMode() });
+        if (m_sourceImage)
```

Comment:
The change log claims that this is being "catched from" an existing test. What test? Why isn’t the test failing? Where is the test expectation change where we now expect it to pass?